src/event_watcher/windows.rs

```diff
@@ -116,7 +143,10 @@ unsafe extern "system" fn wndproc(
     wparam: WPARAM,
     lparam: LPARAM,
 ) -> LRESULT {
-    if let Some(window_data) = window.get_user_data::<WindowData>() {
+    let window_data_ptr = GLOBAL_WINDOW_DATA.load(Ordering::SeqCst);
```

Comment:
I had to change this because the win_event_hook_proc I added doesn't have a handle to our window.